tests/test_wsireader.py

```diff
@@ -979,7 +979,7 @@ def test_VFReader():
 def test_VFReader_read_bounds():
```

Comment:
same as above "VirtualWSIReader"

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/8387736e723a92110cc719baf0490d46a6518d82